tests/runtest.cmd

```diff
@@ -191,6 +192,23 @@ if errorlevel 1 (
     exit /b 1
 )
 
+if not defined %__PerfTests% (
+	exit /b 1
+)
```

Comment:
now with your this change on functional runs I will not get the success message that gets printed when all tests pass successfully. Look at messages at line 219.